src/InnoSetup/Languages/IdfToolsSetup_en-US.isl

```diff
@@ -150,3 +150,5 @@ SummaryComponents=Components
 SummaryDrivers=Drivers
 SummaryTargets=Targets
 SummaryOptimization=Optimization
+ComponentToitJaguar=Toit language - tool Jaguar (beta - jag.exe)
```

Comment:
Perhaps: 

  Toit language support - Jaguar live reloading tool

or something to tell a little bit about what Jaguar is.